specification/monitor/data-plane/readme.md

```diff
@@ -0,0 +1,147 @@
+# MonitorClient
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for MonitorClient.
+
+
+
+---
+## Getting Started
+To build the SDK for MonitorClient, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+---
+
+## Configuration
+
+
+
+### Basic Information
+These are the global settings for the MonitorClient API.
+
+``` yaml
+title: MonitorClient
+description: Monitor Management Client
+openapi-type: arm
+
+tag: package-2018-09
+
+```
+### Tag: package-2018-09
+
+These settings apply only when `--tag=package-2018-09` is specified on the command line.
+
+``` yaml $(tag) == 'package-2018-09'
+input-file:
+- data-plane/preview/2018-09-01-preview/azureMonitorCustomMetricsIngestionApi.json
+```
+
+# Code Generation
+
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+``` yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-python
+  - repo: azure-sdk-for-java
+  - repo: azure-sdk-for-go
+  - repo: azure-sdk-for-node
+  - repo: azure-sdk-for-ruby
+    after_scripts:
+      - bundle install && rake arm:regen_all_profiles['azure_mgmt_monitor']
+```
+
+
+## C#
+
+These settings apply only when `--csharp` is specified on the command line.
+Please also specify `--csharp-sdks-folder=<path to "SDKs" directory of your azure-sdk-for-net clone>`.
+
+``` yaml $(csharp)
+csharp:
+  azure-arm: true
+  payload-flattening-threshold: 1
+  license-header: MICROSOFT_MIT_NO_VERSION
+  namespace: Microsoft.Azure.Management.Monitor
```

Comment:
Is this the correct namespace for the C# implementation?  @shahabhijeet can you review this part?

Reply:
Given that this is data-plane I would think the word `Management` doesn't belong in the namespace but I'm not really sure.